src/ccl_haloprofile.c

```diff
@@ -0,0 +1,134 @@
+#include "ccl_utils.h"
+#include "math.h"
+#include "stdio.h"
```

Comment:
Use include <> for stdlib headers and gsl.
Also, ccl.h already included most of the other ccl_headers. Follow the example in ccl_halomod.c.

Reply:
Thank you! I'll look into it and make these changes in two or three days.